### ISSUE TITLE
NPCs massive spawn at halaa

### ISSUE THREAD

**Issue by lsn371** [dalaranwow/dalaran-wow#4442]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: NPCs massive spawn at halaa might affect the server, basically the guards / ncps are spawning too much there's like 18 units of each npc and it seems they keep spawning, don't know with which frequency

**Expected behaviour**: should only spawn once, and after all the guards are dead and halaa is taken by either faction, they should despawn being replaced with the ones of the faction that took the zone

**Steps to reproduce the problem**:

1. go to halaa
2. watch the npcs
3. wonder about them

**Include proofs for this behaviour**

http://imgur.com/W76fR3i

http://imgur.com/5pP0jFL

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like http://wotlk.openwow.com or http://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Rushor** [comment]
duplicate of https://github.com/dalaranwow/dalaran-wow/issues/3346
